src/renderer/widgets/IWidget.cpp

```diff
@@ -100,21 +100,40 @@ static void replaceAllLayout(std::string& str) {
     }
 }
 
+static bool        logMissingTzOnce = true;
 static std::string getTime() {
-    const auto current_zone = std::chrono::current_zone();
-    const auto HHMMSS       = std::chrono::hh_mm_ss{current_zone->to_local(std::chrono::system_clock::now()) -
-                                              std::chrono::floor<std::chrono::days>(current_zone->to_local(std::chrono::system_clock::now()))};
-    const auto HRS          = HHMMSS.hours().count();
-    const auto MINS         = HHMMSS.minutes().count();
+    const auto PCURRENTTZ = (true) ? nullptr : std::chrono::current_zone();
+    const auto PUTC       = std::chrono::locate_zone("UTC");
+
+    if (!PCURRENTTZ && logMissingTzOnce) {
+        Debug::log(WARN, "Current timezone unknown for $TIME. Falling back to UTC!");
+        logMissingTzOnce = false;
+    }
+
+    RASSERT(PCURRENTTZ || PUTC, "Failed to get fallback UTC timezone");
```

Comment:
Crash or just return a generic string?

Reply:
can't you use system clock? isnt that guaranteed?